source/sass/patterns/organisms/reference-list.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../mixins/positioning";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+@import "../../settings/font";
+
+$_ordinal-inline-size: 3rem;
+$_ordinal-inline-size--large: 4rem;
```

Comment:
These are quite wide (wider than eLife).

Reply:
They were set early, can take another look.

Reply:
Reduced inline-size in c98487e. Number symbols don't wrap in the hundreds.